extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -110,7 +140,14 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       {name: 'u_w', value: screen ? screen.width : null},
       {name: 'vis', value: visibilityStateCodes[visibilityState] || '0'},
       {name: 'wgl', value: global['WebGLRenderingContext'] ? '1' : '0'},
-    ], []);
+    ];
+
+    if (prevFmts) {
+      paramList.push({name: 'prev_fmts', value: prevFmts});
+    }
+
+    return googleAdUrl(
```

Comment:
Agreed. Currently, I'm working on pinning down the definitions of these parameters and thinking of ways of how to generate them in A4A. Presumably, some of these won't make sense to include in A4A, e.g., cbv which is defined as "The version of show_ads.js that generated the request."